### PR TITLE
Based on Centos and fixed spacing

### PR DESCRIPTION
**Pull request by kyguy**
### Type of change

_Select the type of your PR_

- Bugfix
- Enhancement / new feature
- Refactoring

### Description

_Please describe your pull request_

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging

### CODE REVIEW

File: docker-images/stunnel-zookeeper/scripts/stunnel_config_generator.sh [hunk 1]
```diff
@@ -3,42 +3,43 @@
...
+			accept=127.0.0.1:$(expr $port \* 10 + $NODE - 1)
+			connect=${PEER}.${BASE_FQDN}:$port
+
+			EOF
 		fi
 		let NODE=NODE+1
     done
 
-    # Zookeeper port where stunnel forwards received traffic
+	# Zookeeper port where stunnel forwards received traffic
```
Comment (tombentley) [inline docker-images/stunnel-zookeeper/scripts/stunnel_config_generator.sh]: It looks like there's mixed indentation (spaces and tabs) in this file.

**Comment by scholzj** [comment]
Why is the image named stunnel-zookeeper? _Nobody_ interested in Kafa cares about STunnel ... they know Kafka and Zookeeper. Wouldn't it be better to name it `zookeeper-stunnel`?

**Comment by ppatierno** [comment]
We followed the pattern "init-kafka" and not "kafka-init"

**Comment by scholzj** [comment]
Yeah ... I think that's wrong as well ... but I missed it and now I do not want to invest the time into it.

**Comment by ppatierno** [comment]
Agree, even because I don't think that it is so important this kind of change.

**Pull request merged**